diagnostic.md

```diff
@@ -50,22 +46,15 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+relative. Users shows that it is on the user's computer.
```

Comment:
This is an absolute path because the first "/" indicates the root of the system.